.github/workflows/beta-release.yml

```diff
@@ -0,0 +1,59 @@
+name: Release & Publish
+
+on:
+  push:
+    paths-ignore:
+      - 'docs/**'
+      - 'README.md'
+      - 'LICENSE'
+      - '.editorconfig'
+    branches:
+      - main
```

Comment:
How did you test that the action does what you want?
We may add this feature branch to the list to do a test-beta-release and can then remove the branch again.